docs/reference/contributing/guidelines/workflow.md

```diff
@@ -60,6 +60,14 @@ Pull requests on GitHub have to meet the following requirements to keep the code
 - Avoid merging commmits. (Always rebase when possible.)
 - Pull requests should fix a bug, add a feature or refactor.
 
+#### Mbed OS release versioning
+
+Version number MAJOR.FEATURE.PATCH where:
+
+- MAJOR version for incompatible API changes
+- FEATURE version for adding functionality in backward-compatible manner
```

Comment:
Interesting. So we're abandoning the `MINOR` release terminology?

Reply:
"functionality in backward-compatible manner" to "functionality in a backwards-compatible manner"

Reply:
Yes, we have feature releases.

language will fix

Reply:
Marketing strikes again!

Reply:
One of my least favorite "we renamed a standard thing" instances"